lib/ansible/modules/net_tools/infinity/infinity.py

```diff
@@ -1,126 +1,112 @@
 #!/usr/bin/python
 # -*- coding: utf-8 -*-
-#
-# Copyright (c), meiliu@fusionlayer.com, 2017
-#
-# This file is part of Ansible
-#
-# Ansible is free software: you can redistribute it and/or modify
-# it under the terms of the GNU General Public License as published by
-# the Free Software Foundation, either version 3 of the License, or
-# (at your option) any later version.
+
+# Copyright: (c) 2017, <meiliu@fusionlayer.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
 
 from __future__ import (absolute_import, division, print_function)
 __metaclass__ = type
 
-
 ANSIBLE_METADATA = {'metadata_version': '1.1',
                     'status': ['preview'],
                     'supported_by': 'community'}
 
-
-DOCUMENTATION = """
+DOCUMENTATION = r'''
 module: infinity
-short_description: "manage Infinity IPAM using Rest API"
+short_description: Manage Infinity IPAM using Rest API
 description:
-  - "Manage Infinity IPAM using REST API"
+  - Manage Infinity IPAM using REST API.
 version_added: "2.4"
 author:
-  - "Meirong Liu (@MeganLiu)"
+  - Meirong Liu (@MeganLiu)
 options:
   server_ip:
     description:
-      - Infinity server_ip with IP address
+      - Infinity server_ip with IP address.
+    type: str
     required: true
   username:
     description:
-      - Username to access Infinity
-      - The user must have Rest API privileges
+      - Username to access Infinity.
+      - The user must have REST API privileges.
+    type: str
     required: true
   password:
     description:
-      - Infinity password
+      - Infinity password.
+    type: str
     required: true
   action:
     description:
       - Action to perform
+    type: str
     required: true
-    choices:
-      - reserve_next_available_ip
-      - release_ip
-      - delete_network
-      - add_network
-      - reserve_network
-      - release_network
-      - get_network_id
-
+    choices: [add_network, delete_network, get_network, get_network_id, release_ip, release_network, reserve_network, reserve_next_available_ip ]
```

Comment:
Nice catch adding `get_network` as well as `get_network_id`.

Reply:
That's what I was fixing as part of fixing the validate-modules sanity issues.

These were known issues that we simply ignored because nobody took the time to look into them.